gui/main.js

```diff
@@ -2,302 +2,75 @@
 
 require('babel-polyfill')
 
-const AutoLaunch = require('auto-launch-patched')
-const childProcess = require('child_process')
 const Desktop = require('cozy-desktop').default
-const electron = require('electron')
 const notify = require('electron-main-notification')
-const fs = require('fs')
-const lnk = require('lnk')
+
 const debounce = require('lodash.debounce')
-const os = require('os')
 const path = require('path')
-const url = require('url')
-// const uuid = require('node-uuid')
 
-const {spawn} = childProcess
-const {app, BrowserWindow, dialog, ipcMain, Menu, shell, session} = electron
-const autoUpdater = require('electron-updater').autoUpdater
-const autoLauncher = new AutoLaunch({
-  name: 'Cozy-Desktop',
-  isHidden: true
-})
+const autoLaunch = require('./src/main/autolaunch')
```

Comment:
:+:1 it's cool to have finally split `main.js`. It started as a small file, and then added a function, the next time an if/else to handle a special condition, and after that put another entry in a menu, etc. The file grows and grows but it was never a big change that justify to take time to refactor... until now! Yipee!

Reply:
Well, as discussed with @aenario yesterday, last time I tried to split it, it broke electron-builder... 😭